resources/public/api/conf/1.0/application.yaml

```diff
@@ -1,6 +1,6 @@
 openapi: "3.0.3"
 info:
-  title: Tax-free childcare integration API
+  title: Tax-free childcare payment integration API
```

Comment:
please change to payments to match the acual endpoints uri we use

Reply:
also, I think is all camel-case to match other areas. it may be worth double checking